src/main/java/analysis/bachelorarbeit/PopulationHomeActivityFilter.java

```diff
@@ -0,0 +1,68 @@
+package analysis.bachelorarbeit;
```

Comment:
again, this is not part of the repository. Or, if we indeed need it, move it to another package.